### ISSUE TITLE
Documentation for System.Collections.Generic.CollectionExtensions

### ISSUE THREAD

**Issue by orobert91** [dotnet/dotnet-api-docs#3270]
Documentation for System.Collections.Generic.CollectionExtensions should be added:

https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.collectionextensions.tryadd and other methods as well.

https://github.com/dotnet/corefx/issues/17918

**Comment by mairaw** [comment]
Thanks @orobert91. @carlossanlop is this one on your radar to get documented?

**Comment by carlossanlop** [comment]
Yes, it's in the list of 2.0 APIs that my team will document in the next months.
CC @safern , the area owner of Collections.
